ap/tasks/selection.py

```diff
@@ -20,23 +20,31 @@ class SelectEvents(DatasetTask, CalibratorsSelectorMixin, law.LocalWorkflow, HTC
 
     sandbox = dev_sandbox("bash::$AP_BASE/sandboxes/venv_columnar.sh")
 
-    shifts = CalibrateEvents.shifts | {"jec_up", "jec_down"}
+    shifts = {CalibrateEvents, "jec_up", "jec_down"}
```

Comment:
This should probably be removed and injected dynamically by the JEC calibrator (already done in PR #35)

Reply:
Yep. I would leave it to #35 to remove that :)